app/models/alaveteli_pro/plan_with_tax.rb

```diff
@@ -0,0 +1,22 @@
+# -*- encoding : utf-8 -*-
+#
+# Calculate amount + 20% tax for a Stripe::Plan
+#
+# Example
+#
+#   plan = Stripe::Plan.retrieve('pro')
+#   @plan = PlanWithTax.new(plan)
+#   @plan.amount
+#   # => 833
+#   @plan.amount_with_tax
+#   # => 1000
+class AlaveteliPro::PlanWithTax < SimpleDelegator
+  TAX_RATE = BigDecimal('0.20').freeze
+
+  def amount_with_tax
+    net = BigDecimal(amount * 0.01, 0).round(2)
```

Comment:
it looks (from the spec failure) like this will fail on Ruby 2.0.0 unless called as

`BigDecimal.new(amount * 0.01, 0).round(2)`

but the `TAX_RATE` assignment appears to be fine as-is o_0